src/i18n/en.json

```diff
@@ -239,5 +239,10 @@
   "External": "External",
   "Do you really want to remove this media?": "Do you really want to remove this media?",
   "Media deleted": "Media deleted",
-  "Error while deleting media. Try again later...": "Error while deleting media. Try again later..."
+  "Error while deleting media. Try again later...": "Error while deleting media. Try again later...",
+  "Number of players": "Number of players",
+  "Duration (mins)": "Duration (mins)",
+  "Age": "Age",
+  "French": "French",
```

Comment:
There is already translation for lang (`fr` and `en` keys).

Reply:
That's for the language filter label. I'm not sure that's better to change the key in the language filter component to use "fr"/"en" to optimise in this translation file, I prefer to keep it this way:

```
<label htmlFor="french-filter">{t("French")}</label>
```

and not
```
<label htmlFor="french-filter">{t("fr")}</label>
```

Reply:
I didn't tell you to use the previous keys ;-) you're right indeed, full key is more readable. Can you update the key in other files to avoid duplication instead? It not an optimisation issue but more a maintainability one. It's easy to change one key and forget the other if we have two keys for the same word.

Reply:
Same for `Number of player` and `Duration`. You can keep the key you want, but I think avoiding duplicate is important.